students/unsan-Jeong/users/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from users.views import SignUp
+
+urlpatterns = [
+    path('', SignUp.as_view()),
```

Comment:
```suggestion
    path('/siginup', SignUp.as_view()),
```
다음과 같이 url을 설정하면 이 api가 어떤 기능을 하는지 바로 알 수 있습니다.